src/pocketmine/block/Cactus.php

```diff
@@ -127,4 +129,14 @@ public function place(Item $item, Block $blockReplace, Block $blockClicked, int
 
 		return false;
 	}
+
+	public function canBlockLand(Fallable $block) : int{
+		$parentDecision = parent::canBlockLand($block);
+		if ($parentDecision == FallingBlock::STATE_DROP_ITEM) {
+			return $parentDecision;
+		}
+
+		return FallingBlock::STATE_DROP_ITEM;
```

Comment:
The logic in this function seems redundant. It seems like this could be reduced to a single line.

Reply:
This problem has not been resolved.